spyder/widgets/projects/explorer.py

```diff
@@ -169,6 +169,7 @@ def delete(self, fnames=None):
 
 class ProjectExplorerWidget(QWidget):
     """Project Explorer"""
+    redirect_stdio = Signal(bool)

```

Comment:
Is this signal necessary?

Reply:
Yep, here the reason: https://github.com/spyder-ide/spyder/blob/3.x/spyder/widgets/explorer.py#L703